server/controllers/api/getlist.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = function(app) {
+	
+	var bodyParser = require('body-parser')
+	var jsonParser = bodyParser.json()
+	var urlencodedParser = bodyParser.urlencoded({ extended: false })
+
+	app.use(bodyParser.urlencoded({ extended: false }))
+	app.use(bodyParser.json())
+	
+	app.post('/api/getlist', function (req, res) {
+		console.log('inside post');
+  		if (!req.body) return res.sendStatus(400);
+
+  		var requested = req.body;
+
+  		res.header("Access-Control-Allow-Origin", "*");
+
+		app.db.select([
```

Comment:
If we return all of these, doesn't that make https://github.com/Learn-by-doing/hobobar/issues/6 obsolete?